### PR TITLE
note on office hours

### PR DESCRIPTION
**Pull request by drewwells**
The zoom meeting requires people login to zoom and there's a waiting room. That should be sufficient safeguards for unintended guests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # konk
 
+
+Office Hour 11am CST on Fridays landing on even days [https://infoblox.zoom.us/j/98130157567](https://infoblox.zoom.us/j/98130157567)
```
Comment (kd7lxl) [inline README.md:4]: I'm not a an of the placement. I feel like this doesn't need to be the very first thing in the readme. Maybe put it after the description.

At a minimum, we should advertise in UTC. CST is a bit obscure. I wonder if there's a tool we could link that would place this time on a chart showing local times...

Comment (kd7lxl) [reply README.md:4]: https://www.worldtimebuddy.com/?qm=1&lid=100,30,8,6&h=100&date=2020-11-19&sln=17-17.5&hf=0

Comment (drewwells) [reply README.md:4]: how about tv time, 8AM PST/12 EST

Comment (kd7lxl) [reply README.md]: Good enough.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # konk
 
+
```
Comment (kd7lxl) [inline README.md:3]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -8,6 +9,9 @@ konk does not start a kubelet and therefore does not support any resources that
 
 This repo provides a konk helm chart that can be used to deploy an instance of konk with helm, and a konk-operator that watches for konk CRs and will deploy a konk instance for each of them.
 
+
```
Comment (kd7lxl) [inline README.md:12]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -8,6 +8,8 @@ konk does not start a kubelet and therefore does not support any resources that
 
 This repo provides a konk helm chart that can be used to deploy an instance of konk with helm, and a konk-operator that watches for konk CRs and will deploy a konk instance for each of them.
 
+Office Hour 9am PST/12pm EST on Fridays landing on even days [https://infoblox.zoom.us/j/98130157567](https://infoblox.zoom.us/j/98130157567)
```
Comment (kd7lxl) [inline README.md:11]:
```suggestion
## Office Hour

Office Hour 9am PST/12pm EST on Fridays landing on even days [https://infoblox.zoom.us/j/98130157567](https://infoblox.zoom.us/j/98130157567)
```

**Review by kd7lxl — APPROVED** [review]

**Pull request merged**